backend/alembic/versions/6d69756a1476_add_user_settings.py

```diff
@@ -0,0 +1,31 @@
+"""Add user settings
+
+Revision ID: 6d69756a1476
+Revises: 729cd1e8cde1
+Create Date: 2018-05-10 16:12:06.872006
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6d69756a1476'
+down_revision = '729cd1e8cde1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Apart of creating new table, we should also take care about existing users. What will happen to them?

I've applied your migration to my local environment where I had one user in the DB. Then, I've tried to skip my tutorial and... I got empty settings (set to None), so skipping failed :(

To overcome this issue, please create settings for each user that already exists in the database. To do so, you can use INSERT ... SELECT query. In case of troubles please contact with me or Karolina directly.